spec/features/task_managements_spec.rb

```diff
@@ -0,0 +1,29 @@
+require "rails_helper"
+
+RSpec.feature "TaskLogs", type: :feature do
+  before do
+    Capybara.default_driver = :selenium
+    workdey_data = Seed.new
+    workdey_data.create_all
+    create(:task_management, task_desc: "Iron these clothes for me")
+  end
+
+  scenario "should see a list of tasks created if you are a tasker" do
+    log_in_with "temitope.amodu@andela.com", "1234567890"
+    expect(page).to have_content("Taskee Name")
```

Comment:
Can we test to confirm if the page actually loads the tasks created by this tasker? You can create mock tasks for this tasker and confirm the page renders only this tasker's tasks.